build.gradle

```diff
@@ -516,19 +516,20 @@ subprojects {
         }
 
         violationRules {
-          minBranches = project.ext.has('diffCoverageThreshold') ? project.ext.diffCoverageThreshold : 0.40
+          minBranches = project.ext.has('diffCoverageThreshold') ? project.ext.diffCoverageThreshold : 0.45
           failOnViolation = true
         }
       }
 
-      task logDiffCoverage {
+      task logCoverage {
         doLast {
+          parseJacocoXml("$buildDir/reports/jacoco/test/jacocoTestReport.xml")
```

Comment:
Just curious, what does this do?

Reply:
The diff coverage is mostly useful in highlighting the coverage gaps introduced by the changes as part of the PR.
My guess is, this provides the state of coverage for the entire code base based on our tests.

Reply:
Yes, that's correct. `test/jacocoTestReport.xml` tells us the overall status for that module and `diffCoverage/test.xml` tells the coverage for the diff.

Since we don't integrate with some code coverage github action yet (since most of they only support line coverage), I figure it might be good to print out the overall status for people to take a look if they are interested.

Reply:
Regarding making the code cov easier to browse online, I was thinking we could have a separate repo to host the jacoco reports, and push these files to that other repo as part of the publish step, then that other repo could render those report files. Just a random idea (out of scope of this PR, obviously...)...